packages/commonwealth/test/unit/rules/rolesHelpers.spec.ts

```diff
@@ -24,34 +24,7 @@ describe('getHighestRole() unit tests', () => {
         deny: BigInt(0),
       },
     ];
-    const role = await getHighestRole(roles);
+    const role = await getHighestRoleFromCommunityRoles(roles);
     assert.equal(role.name, 'admin');
   });
 });
-
-describe('getHighestRole() unit tests', () => {
```

Comment:
Why did you remove the unit test?

Reply:
it was a duplicate test, I accidentally copied it